### PR TITLE
[Templates] Fix 'ghost alert banners' on Step By Step layout

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR fixes an issue where grey boxes were showing up on the Step by Step template when the selection of "none" is chosen by a CMS writer for the alert box. This PR fixes it by using the `alert_single` fragment instead of the standard alert paragraph.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/15232
## Testing done
Tested against http://localhost:3001/preview?nodeId=8520

### Before
image

### After
The image doesn't show but the point is that the gray box is now gone.

image

## Acceptance criteria
- [ ] No more "ghost alerts"

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/includes/related-benefit-hubs.drupal.liquid [hunk 1]
```diff
@@ -12,8 +12,12 @@
           <!-- Show the link if it hasn't been rendered yet -->
           {% unless supportServiceIDs contains supportService.entity.entityId %}
             <li>
-              <p class="vads-u-margin-top--0"><strong>{{ supportService.entity.title }}</strong> <a href="{{ supportService.entity.fieldLink.url.path }}"
-                  rel="noreferrer noopener">{{ supportService.entity.fieldPhoneNumber }}</a></p>
+              <p class="vads-u-margin-top--0"><strong>{{ supportService.entity.title }}</strong>
+              {% if supportService.entity.fieldLink %}
+                <a href="{{ supportService.entity.fieldLink.url.path }}"
+                    rel="noreferrer noopener">{{ supportService.entity.fieldPhoneNumber }}</a>
+              {% endif %}
```
Comment (ncksllvn) [inline src/site/includes/related-benefit-hubs.drupal.liquid:19]: This if-statement fixes an Axe error where the 711 message was rendering an empty <a> element.

image

File: src/site/layouts/step_by_step.drupal.liquid [hunk 2]
```diff
@@ -60,22 +60,7 @@
...
-                        {% endif %}
-                      {% endfor %}
-                    {% else %}
-                      {% if fieldStep.entity.fieldAlert.entity != empty %}
-                        <div class="vads-u-margin-bottom--4">
-                          {% include "src/site/blocks/alert.drupal.liquid" with alert = fieldStep.entity.fieldAlert.entity %}
-                        </div>
-                      {% endif %}
-                    {% endif %}
+                    {% include "src/site/paragraphs/alert_single.drupal.liquid" with alert = fieldStep.entity.fieldAlert.entity %}
```
Comment (kelsonic) [inline src/site/layouts/step_by_step.drupal.liquid:63]: Hmm, what if there are multiple alerts? Looks like the previous code was trying to support that but not sure if it matters

**Review by kelsonic — APPROVED** [review]
LGTM! One clarifying comment 🙂

File: src/site/layouts/step_by_step.drupal.liquid (hunk 2 above)
Comment (ncksllvn) [reply src/site/layouts/step_by_step.drupal.liquid:63]: I'm not sure - so many data shapes. But the `alert_single` matches the `entityBundle` for the `fieldStep.entity.fieldAlert.entity` so I think this is correct

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**